locksmith/src/controllers/v2/keyController.ts

```diff
@@ -58,6 +63,72 @@ export default class KeyController {
     }
   }
 
+  /**
+   * Initiates worker task to get all the keys for a given lock
+   */
+  async startKeyJob(request: Request, response: Response) {
+    try {
+      const jobId = randomUUID()
+
+      const payload = LockKeyJobPayload.parse({
+        jobId: jobId as string,
+        lockAddress: Normalizer.ethereumAddress(request.params.lockAddress),
+        network: Number(request.params.network),
+        query: request.query.query ?? '',
+        filterKey: request.query.filterKey,
+        expiration: request.query.expiration ?? 'active',
+        approval: request.query.approval ?? 'minted',
+        loggedInUserAddress: Normalizer.ethereumAddress(
+          request.user!.walletAddress
+        ),
+      })
+
+      const workerUtils = await WorkerUtilsSingleton.getInstance()
```

Comment:
I am not sure this is required. Please use `quickAddJob`.

Reply:
So I need to remove the Singleton Instance for worker Utils?